Regression on ROS 2 Buildfarm for Rolling

`polygon_rviz_plugins 1.1.0` is failing to build on the buildfarm with the error. This is blocking the [announced Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2025-04-11/43166). We will proceed with the sync which will remove these binaries from Rolling but they will be added once the build is fixed and a new release is bloomed.

```
23:14:46 CMake Error at CMakeLists.txt:19 (find_package):
23:14:46   By not providing "Findament_cmake_ros.cmake" in CMAKE_MODULE_PATH this
23:14:46   project has asked CMake to find a package configuration file provided by
23:14:46   "ament_cmake_ros", but CMake did not find one.
23:14:46 
23:14:46   Could not find a package configuration file provided by "ament_cmake_ros"
23:14:46   with any of the following names:
23:14:46 
23:14:46     ament_cmake_rosConfig.cmake
23:14:46     ament_cmake_ros-config.cmake
23:14:46 
23:14:46   Add the installation prefix of "ament_cmake_ros" to CMAKE_PREFIX_PATH or
23:14:46   set "ament_cmake_ros_DIR" to a directory containing one of the above files.
23:14:46   If "ament_cmake_ros" provides a separate development package or SDK, be
23:14:46   sure it has been installed.
```


https://build.ros2.org/view/Rbin_uN64/job/Rbin_uN64__polygon_rviz_plugins__ubuntu_noble_amd64__binary/80/consoleFull

--- a/polygon_rviz_plugins/package.xml
+++ b/polygon_rviz_plugins/package.xml
@@ -7,7 +7,7 @@
   <maintainer email="davidvlu@gmail.com">David V. Lu!!</maintainer>
   <license>BSD</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
   <depend>color_util</depend>
   <depend>geometry_msgs</depend>
   <depend>pluginlib</depend>

